src/Faker/Provider/Base.php

```diff
@@ -168,8 +168,19 @@ public static function randomAscii()
      *
      * @return array New array with $count elements from $array
      */
-    public static function randomElements(array $array = array('a', 'b', 'c'), $count = 1, $allowDuplicates = false)
+    public static function randomElements($array = array('a', 'b', 'c'), $count = 1, $allowDuplicates = false)
     {
+        if ($array instanceof \Traversable) {
+
+            $return = array();
+
+            foreach ($array as $element) {
+                $return[] = $element;
+            }
+
+            $array = $return;
```

Comment:
bad practice to update a parameter. Prefer using a new variable with a ternary expression